addons/survey/views/survey_templates_statistics.xml

```diff
@@ -194,7 +193,7 @@
                         </tr>
                     </thead>
                     <tbody>
-                        <tr t-foreach="question_data['numerical_common_lines']" t-as="common_line">
+                        <tr t-foreach="question_data['common_lines']" t-as="common_line">
                             <td><span t-esc="common_line[0]"></span></td>
```

Comment:
Looks like it is passing a date value correctly, but the t-esc was showing the server date format instead of the user date format (or however it's capturing the standard date format, I can't quite figure out how to test it with different language settings since it appears to default to MM/DD/YYYY even a user has their language set to French(BE)). Unfortunately this means I needed to add yet another t-if statement for checking the question_type so that `t-options='{"widget": "date"}'` is applied appropriately.